agvis/static/js/ConfigControl.js

```diff
@@ -397,11 +473,14 @@ function addSidebarConfig(win, options, sidebar) {
             dt = dt.toUTCString();
 
             document.cookie = `opacity${win.num}=${val};expires=${dt};path=/`;
-
-            win.legend.update();
```

Comment:
This change removed a line that was accidentally added when making the legend bar.